sky/services/editing/ios/keyboard_impl.mm

```diff
@@ -134,17 +137,18 @@ - (void)deleteBackward {
     : binding_(this, request.Pass()), client_([[KeyboardClient alloc] init]) {}
 
 KeyboardImpl::~KeyboardImpl() {
+  [client_ hide];
```

Comment:
Is this call necessary?  Is releasing the view sufficient to remove it from the parent?